examples/gltf/app.js

```diff
@@ -0,0 +1,86 @@
+import {GLTFParser} from '@loaders.gl/gltf';
+import GL from '@luma.gl/constants';
+import {AnimationLoop, Framebuffer, Cube, setParameters, clear, GLTFInstantiator, Texture2D} from 'luma.gl';
+import {Matrix4, radians} from 'math.gl';
+
+// const GLTF_URL = "https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Models/master/2.0/2CylinderEngine/glTF-Binary/2CylinderEngine.glb";
+const GLTF_URL = "https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Models/master/2.0/Avocado/glTF-Binary/Avocado.glb";
+
+const INFO_HTML = `
+<p>
+  <b>glTF</b> rendering.
+<p>
+A luma.gl <code>glTF</code> renderer.
+`;
+
+export const animationLoopOptions = {
+
+  onInitialize: ({gl}) => {
+    const models = [];
+
+    const gltfParser = new GLTFParser();
+    fetch(GLTF_URL).then(res => res.arrayBuffer()).then(data => {
+      gltfParser.parse(data);
+      const instantiator = new GLTFInstantiator(gl, {
+        getImage: index => {
```

Comment:
Shouldn't this image handling be built in to the GLTFInstantiator, rather than in the example?

Reply:
So what should the parameters of GLTFInstantiator be?

Reply:
> So what should the parameters of GLTFInstantiator be?

I guess we'll discover that as we complete the implementation. In general, the fewer options the better.